applications/integrationtests/mechanics/StructuralGradientDamage.cpp

```diff
@@ -36,19 +35,19 @@ BOOST_AUTO_TEST_CASE(Integrand)
     double ft = 4;
     double fc = 40;
     double gf = 0.021;
-    double c = 1.25;
+    double L = 40;
+    double c = 0.25;
 
     double k0 = ft / E;
     Laws::LinearElastic<1> elasticLaw(E, nu);
-    Constitutive::DamageLawExponential dmg(k0, ft / gf, 1.);
+    Constitutive::DamageLawExponential dmg(k0, ft / gf / 10, 1.);
     Constitutive::ModifiedMisesStrainNorm<1> strainNorm(nu, fc / ft);
 
-    using Gdm = Integrands::GradientDamage<1, Constitutive::DamageLawExponential>;
+    using Gdm = Integrands::GradientDamage<1, Constitutive::DamageLawExponential, Integrands::DecreasingInteraction>;
```

Comment:
Could that live in a namespace or something? Seems quite specific, while `Integrands::DecreasingInteraction` sounds very generic.